onnx/large_helper.py

```diff
@@ -0,0 +1,333 @@
+# Copyright (c) ONNX Project Contributors
```

Comment:
nit: Reminder to add a summary docstring for this file

Reply:
Done. I looked into other files. There are not so many files defining a dcostring for the module and the comment is not really helpful.

Reply:
The summary from experience will help clarify the scope of the module and help us decide when to put things in a new module down the line

Reply:
done